conans/client/cmake.py

```diff
@@ -330,6 +340,11 @@ def _build_new(self, args=None, build_dir=None, target=None):
         command = "cmake --build %s" % arg_list
         self._conanfile.run(command)
 
+    def package(self):
```

Comment:
configure => configures the project (by the way, this is not a cmake command, configure is conan naming)
build => build
test => run tests
install => create package. It is not "installing". If ``package_folder`` is not defined, it will fail (so I don't allow system-wide installs, which are the default for ``cmake install``)